### ISSUE TITLE
Breakage with Ruby 2.5

### ISSUE THREAD

**Issue by rjp** [localytics/humidifier#78]
Recently refreshed my system which brought me Ruby 2.5 by default and humidifier stopped working. A bare stack works but adding any resource (ie `Humidifier::ApiGateway::Model` or `Humidifier::DynamoDB::Table`) causes the breakage below. The exact same stack will work fine on Ruby 2.4.

The specific Ruby versions are:
```
[rjp@scw-21bc77 cfn]$ ruby -v
ruby 2.5.0p0 (2017-12-25 revision 61468) [armv7l-linux-eabihf]

[rjp@scw-21bc77 cfn]$ ruby-2.4 -v
ruby 2.4.3p205 (2017-12-14 revision 61247) [armv7l-linux-eabihf]
```

```
Traceback (most recent call last):
        14: from dynamodb.rb:14:in `<main>'
        13: from /home/rjp/.gem/ruby/2.5.0/gems/humidifier-1.12.0/lib/humidifier/stack.rb:44:in `to_cf'
        12: from /home/rjp/.gem/ruby/2.5.0/gems/humidifier-1.12.0/lib/humidifier/stack.rb:77:in `enumerable_resources'
        11: from /home/rjp/.gem/ruby/2.5.0/gems/humidifier-1.12.0/lib/humidifier/stack.rb:77:in `each_with_object'
        10: from /home/rjp/.gem/ruby/2.5.0/gems/humidifier-1.12.0/lib/humidifier/stack.rb:77:in `each'
         9: from /home/rjp/.gem/ruby/2.5.0/gems/humidifier-1.12.0/lib/humidifier/stack.rb:82:in `block in enumerable_resources'
         8: from /home/rjp/.gem/ruby/2.5.0/gems/humidifier-1.12.0/lib/humidifier/stack.rb:82:in `map'
         7: from /home/rjp/.gem/ruby/2.5.0/gems/humidifier-1.12.0/lib/humidifier/stack.rb:82:in `each'
         6: from /home/rjp/.gem/ruby/2.5.0/gems/humidifier-1.12.0/lib/humidifier/stack.rb:83:in `block (2 levels) in enumerable_resources'
         5: from /home/rjp/.gem/ruby/2.5.0/gems/humidifier-1.12.0/lib/humidifier/resource.rb:39:in `to_cf'
         4: from /home/rjp/.gem/ruby/2.5.0/gems/humidifier-1.12.0/lib/humidifier/resource.rb:39:in `map'
         3: from /home/rjp/.gem/ruby/2.5.0/gems/humidifier-1.12.0/lib/humidifier/resource.rb:39:in `each'
         2: from /home/rjp/.gem/ruby/2.5.0/gems/humidifier-1.12.0/lib/humidifier/resource.rb:40:in `block in to_cf'
         1: from /home/rjp/.gem/ruby/2.5.0/gems/humidifier-1.12.0/lib/humidifier/props/base.rb:30:in `to_cf'
/home/rjp/.gem/ruby/2.5.0/gems/humidifier-1.12.0/lib/humidifier/serializer.rb:11:in `dump': uninitialized constant #<Class:Humidifier::Serializer>::Date (NameError)
```

**Comment by kddeisz** [comment]
@rjp, that's weird. Not sure why that's happening but maybe something changed where previously Date was required and now it's not, from the standard library. I've added an explicit `require 'date'` to a new version, can you test out `1.13.0.1` to see if it fixes it for you?

**Comment by rjp** [comment]
Can confirm that `1.13.0.1` fixes my issue with Ruby 2.5.0p0. Thanks!
